src/main/java/com/tickets/dao/impl/MovieDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.tickets.dao.impl;
+
+import com.tickets.dao.MovieDao;
+import com.tickets.lib.Dao;
+import com.tickets.model.Movie;
+import com.tickets.util.HibernateUtil;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class MovieDaoImpl implements MovieDao {
+    @Override
+    public Movie add(Movie movie) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            Long movieId = (Long) session.save(movie);
+            transaction.commit();
+            movie.setId(movieId);
```

Comment:
check [this](https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_01.html) one more time